### PR TITLE
Implemented payments model

### PR DESCRIPTION
**Pull request by Gilgumish**
1. Added Payment Model
2. Configured Serializers
3. Set Up ViewSet
4. Defined URLs
5. Implemented Model Tests

### CODE REVIEW

File: payment/models.py [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+    ]
+
+    TYPE_CHOICES = [
+        ("PAYMENT", "Payment"),
+        ("FINE", "Fine"),
+    ]
+
+    status = models.CharField(max_length=10, choices=STATUS_CHOICES, default="PENDING")
+    type = models.CharField(max_length=10, choices=TYPE_CHOICES, default="PAYMENT")
+    borrowing = models.ForeignKey(Borrowing, on_delete=models.CASCADE)
```
Comment (oleksandra-shershen) [inline payment/models.py]: Add related_name

File: payment/models.py [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+        ("PENDING", "Pending"),
+        ("PAID", "Paid"),
+    ]
+
+    TYPE_CHOICES = [
+        ("PAYMENT", "Payment"),
+        ("FINE", "Fine"),
+    ]
+
+    status = models.CharField(max_length=10, choices=STATUS_CHOICES, default="PENDING")
```
Comment (oleksandra-shershen) [inline payment/models.py]: max_length must be 7 here

File: payment/models.py [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+    type = models.CharField(max_length=10, choices=TYPE_CHOICES, default="PAYMENT")
+    borrowing = models.ForeignKey(Borrowing, on_delete=models.CASCADE)
+    session_url = models.URLField()
+    session_id = models.CharField(max_length=255)
+    money_to_pay = models.DecimalField(decimal_places=2, max_digits=10)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f"{self.borrowing.book.title} - {self.get_status_display()}"
```
Comment (oleksandra-shershen) [inline payment/models.py:27]: Good idea with `get_status_display()`

File: payment/models.py (hunk 1 above)
Comment (Gilgumish) [reply payment/models.py]: Resolved

File: payment/models.py (hunk 2 above)
Comment (Gilgumish) [reply payment/models.py]: Resolved

File: payment/views.py [hunk 4]
```diff
@@ -1,3 +1,19 @@
...
+class PaymentViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.UpdateModelMixin,
+    viewsets.GenericViewSet,
+):
+    queryset = Payment.objects.select_related("borrowing__book", "borrowing__user")
+
+    def get_serializer_class(self):
```
Comment (oleksandra-shershen) [inline payment/views.py:21]: Please, use same code style in project, for example:
```
    queryset = Borrowing.objects.all().select_related("book", "user")
    serializer_class = BorrowingSerializer

    def get_serializer_class(self):
        if self.action == "list":
            return BorrowingListSerializer
        elif self.action == "retrieve":
            return BorrowingDetailSerializer
        return self.serializer_class
```

Comment (Gilgumish) [reply payment/views.py:21]: Resolved

**Review by Boyok1337 — APPROVED** [review]
LGTM!

**Review by VitalyBashkiser — APPROVED** [review]
Well done 👍

**Pull request merged**